packages/core/src/api/apis/definitions/featureFlags.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import ApiRef from '../ApiRef';
+import { FeatureFlagName } from '../../plugin/types';
+
+/**
+ * The feature flags API is used to toggle functionality to users across plugins and Backstage.
+ *
+ * Plugins can use this API to register feature flags that they have available
+ * for users to enable/disable, and this API will centralize the current user's
+ * state of which feature flags they would like to enable.
+ *
+ * This is ideal for Backstage plugins, as well as your own App, to trial incomplete
+ * or unstable upcoming features. Although there will be a common interface for users
+ * to enable and disable feature flags, this API acts as another way to enable/disable.
+ */
+
+export enum FeatureFlagState {
+  NotEnabled = 0,
```

Comment:
`Disabled`? :D

Reply:
Sure and that's what I had, but I think it's actually less confusing if it's `NotEnabled`. Disabled can be perceived to imply that that either it cannot be enabled.

Reply:
Yeah true, maybe another pair? `Active` and `Inactive`?

Reply:
Not sure if that's any really any different from what I have. `Active` and `Inactive` are states that could be triggered as side-effects (example, I do something else and this "becomes" active). `NotEnabled` and `Enabled` I feel indicates some sort of action needs to be taken.

If others from @spotify/backstage-core still disagree, I'll surrender :P

Reply:
Heh, I am personally fine with either :)

Reply:
Yeah not too fussed either, but haven't seen `NotEnabled` used anywhere else :p Occurrences on GitHub diff by 5 orders of magnitude for JS xP

Reply:
Haha, I can really imagine that :) Wording-feelings aside, I gotta say I've never seen NotEnabled :)

Reply:
Hmm how about `FeatureFlags.On` and `FeatureFlags.Off`?